modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -17,7 +21,7 @@ Similarly, removing an interface does not delete the policy.
 
 .Procedure
 
-. Update the `NodeNetworkConfigurationPolicy` manifest used to create the interface. The following example removes a Linux bridge:
+. Update the `NodeNetworkConfigurationPolicy` manifest used to create the interface. The following example removes a Linux bridge. The policy also configures the `eth1` NIC with DHCP to avoid losing connectivity:
```

Comment:
```suggestion
. Update the `NodeNetworkConfigurationPolicy` manifest used to create the interface. The following example removes a Linux bridge and configures the `eth1` NIC with DHCP to avoid losing connectivity:
```